src/Core/Extensions/ChannelExtensions.cs

```diff
@@ -0,0 +1,12 @@
+using Discord;
+
+namespace BrackeysBot
+{
+    public static class ChannelExtensions
+    {
+        public static string Linkable(this IMessageChannel channel)

```

Comment:
Doesn't `IMessageChannel` implement `IMentionable`?
`channel.Mention` should suffice.

Reply:
'IMessageChannel' does not contain a definition for 'Mention' and the best extension method overload 'UserExtensions.Mention(ulong)' requires a receiver of type 'ulong' [BrackeysBot]

So apparently not.

Reply:
Ah right, that was `ITextChannel`.
Is there a reason to use `IMessageChannel` over `ITextChannel`?

Reply:
Is ITextChannel supported as parameter input of commands?

Reply:
Any implementation of `IChannel` should work.

Reply:
Done.